site/en/docs/extensions/mv3/tut_oauth/index.md

```diff
@@ -104,15 +105,14 @@ Open the Extensions Management page at `chrome://extensions`, ensure developer m
 upload the unpackaged extension directory. Compare the extension ID on the extensions management
 page to the Item ID in the Developer Dashboard. They should match.
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/n6jGhPAAxEjOweiTePjP.png",
-       alt="The ID of the extension matches in all places", height="438", width="567" %}
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/7T1Ko139zBRENzSyKpgB.png", alt="Extension Ids matching", width="350", height="250" %}
```

Comment:
```suggestion
{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/7T1Ko139zBRENzSyKpgB.png",
       alt="Extension Ids matching", width="350", height="250" %}
```

Line break to wrap at 100 characters. I've found myself using this specific styling (`src` attribute aligned with first attribute on the next line) as it generally works well for images that don't have long alt text.